### DISCUSSION TITLE
Cluster NEW LTSP

### DISCUSSION THREAD

**Discussion by CarlosAPCunha** [ltsp/ltsp#275] [Q&A]
HI!!

Today it has a structure with 4 LTSP5 servers (debian 8) on one server (TFTP, Chroots) that has a LTSP cluster manager and another server that does DHCP, total of:

4 Application servers = LTSP
1 TFTP server (Admin Cluster)
1 DHCP server

This works well, but I have problems with updating and sound issues, so I started to migrate to the new LTSP.

My doubt, how to make cluster with new LTSP?

Thank you for your dedication to the LTSP project.

regrads;

**Comment by alkisg** [comment]
Hi, ltsp-cluster hasn't been maintained for many years. But it's easy to do clustering without it.

What are you looking for, a solution for LTSP fat clients or for LTSP thin clients?
What are your client specs, and how many clients do you have?
https://ltsp.org/advanced/specs/

**Comment by CarlosAPCunha** [comment]
Thanks for the quick response.

I need in the scenarios of "thin client" where the applications are executed in the server, because the majority of the "hardware" is of 512M or 1G of Ram, they are very simple stations.

4 Application server with 30G Ram / 6 to 8 Cores and Disk in Storage (SAS).

About 100 simultaneous users, and it works very well, but for the reasons mentioned I am trying to update the process.

regards;

**Comment by alkisg** [comment]
Regarding thin client support, you'll need xrdp or x2go, read this:
https://github.com/ltsp/community/issues/32#issuecomment-541459584

Regarding load balancing, you can just install LTSP in 4 servers; if they're configured in the default proxyDHCP mode, then all of them can boot any client, and you can balance with a little ipxe menu editing.

**Comment by CarlosAPCunha** [comment]
Hi!
  As I understand it, do you recommend using XRDP?

For more information about my scenario in a more complete way?

- Stations in "thin client format" log on to the server via winbind authentication (which 4 application servers are joined to the AD domain), your / home is shared via nfs (I forgot to mention it before, but to this server that stores the user home).

Can I do all of this via XDRP?
Can the user load the screen via IPX already falls into a remote session only if it is necessary to log in (inform user / password)?

The blanking hj occurs by resource in the servers the server that has TFTP connect via port 8000 in the application servers and analyzes load / cpu / ram etc. determined which server the user will be directed, would there be something like that?

Note: Some stations have printers / systems via serial port and ubs would this work? and i need sound too.

Regards and sorry if then much question....

**Comment by alkisg** [comment]
Sure, you can do all these. But, since ltsp-cluster isn't maintained anymore, you'll need to write a few scripts or hire someone to write them for you.

Load balancing can be done either automatically (e.g. mac-address mod number-of-servers) or via a simple RAM/CPU analyzing script.

The new LTSP can help you in netbooting everything. But you'll need to provide the rest of the scripts yourself.

**Comment by CarlosAPCunha** [comment]
Thanks again.

Do you have any "how to" implement LTSP with XRTP? (Thin-clint not fat model)
As you understand it, that would be what you recommend and that would suit me.

Regards

**Comment by alkisg** [comment]
There's some information in the wiki, for example https://github.com/ltsp/community/wiki/RDP-Kiosks-on-Ubuntu.

**Comment by CarlosAPCunha** [comment]
Thanks again, i Will go test.
